Vwitter/Models/Vent.m

```diff
@@ -0,0 +1,28 @@
+//
+//  Vent.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/12/22.
+//
+
+#import "Vent.h"
+
+@implementation Vent
+
+@dynamic author;
+@dynamic ventContent;
+
++ (nonnull NSString *)parseClassName {
+    return @"Vent";
+}
+
+- (instancetype)initWithVentContent:(NSString *)ventContent {
+    if (self = [super init]) {
+        // Initialize self
+        self.author = [PFUser currentUser];
+        self.ventContent = ventContent;
+    }
+    return self;
+}
```

Comment:
To simplify backend queries, the `vent` will probably be one of those classes where, in addition to the pointer column (author), you would want an `authorUserId` column which is just the author's objectId